test/python/test_backends.py

```diff
@@ -100,7 +100,7 @@ def test_local_backend_status(self):
         backend = local_provider.get_backend(name='local_qasm_simulator')
         status = backend.status
         schema_path = self._get_resource_path(
-            'backends/backend_status_schema_py.json', path=Path.SCHEMAS)
+            'old/backends/backend_status_schema_py.json', path=Path.SCHEMAS)
```

Comment:
What about using `v0.1/backends/....` instead of `old/backends/...`? This would allow us to introduce the concept of schema versions (even if in a primitive form) and play nicer with the future changes.

Reply:
As long as you think it will be enough to indicate that we are deprecating these schemas